### PR TITLE
Added note to each assembly stating that Serverless is no longer supported on OCP 4.2

### PR DESCRIPTION
**Pull request by abrennan89**
By request from @sferich888 until it is possible to add a deprecated banner to docs on the customer portal.

### CODE REVIEW

File: serverless/cluster-logging-serverless.adoc [hunk 1]
```diff
@@ -6,6 +6,12 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
+====
+You are viewing documentation for a release of Red Hat OpenShift Serverless is no longer supported.
```
Comment (mgencur) [inline serverless/cluster-logging-serverless.adoc:11]: Red Hat OpenShift Serverless `that` is no longer supported? I'm curious if something is missing in the sentence.

And same comment for all the occurrences of this text.

File: serverless/configuring-knative-serving-autoscaling.adoc [hunk 2]
```diff
@@ -6,6 +6,12 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
+====
+You are viewing documentation for a release of Red Hat OpenShift Serverless is no longer supported.
```
Comment (mgencur) [inline serverless/configuring-knative-serving-autoscaling.adoc:11]: Same comment as above.

**Review by mgencur — COMMENTED** [review]
Just one nit. Otherwise looks ok.

File: serverless/cluster-logging-serverless.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/cluster-logging-serverless.adoc:11]: LOL good thing you read it, I'll add "that" for sure. Thanks!

**Comment by mgencur** [comment]
QE Approved.

**Comment by mburke5678** [comment]
@abrennan89 LGTM

**Pull request merged**